charts/sessionspaces/charts/sessionspaces/templates/clusterrole.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sessionspaces-clusterrole
```

Comment:
We don't need to affix the kind here. Would probably be best just to use the `sessionspaces.fullname`